pr2eus/robot-interface.l

```diff
@@ -160,6 +160,11 @@
      msg))
   (:angle-vector
    (av &optional (tm 3000) (ctype controller-type) (start-time 0))
+   ;; check validity of tm
+   (unless (numberp tm)
+     (warn ";; interpolation time is not a number: ~A~%" tm)
```

Comment:
yes, that would be better.

--
◉ Kei Okada


On Mon, Jan 12, 2015 at 2:18 PM, Ryohei Ueda <notifications@github.com>
wrote:

> In pr2eus/robot-interface.l
> <https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/86#discussion-diff-22775501>
> :
>
> > @@ -160,6 +160,11 @@
> >       msg))
> >    (:angle-vector
> >     (av &optional (tm 3000) (ctype controller-type) (start-time 0))
> > +   ;; check validity of tm
> > +   (unless (numberp tm)
> > +     (warn ";; interpolation time is not a number: ~A~%" tm)
>
> In addition to that, I prefer to use error.
> we should stop program here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/86/files#r22775501>.
>